pkg/logs/client.go

```diff
@@ -144,6 +123,50 @@ func (c *Client) PrintTask(taskID string, file string, opts Options) error {
 	return nil
 }
 
+// FollowTask follows a task's logs.
+func (c *Client) FollowTask(taskID string, file string, print bool, opts Options) error {
+	endpoint := fmt.Sprintf("/system/v1/logs/v2/task/%s/file/%s?cursor=END&skip=%d", taskID, file, opts.Skip)
+
+	client := sse.NewClient(c.http.BaseURL().String() + endpoint)
+	client.Connection = c.http.BaseClient()
+	client.Headers["Authorization"] = c.http.Header().Get("Authorization")
+	client.Headers["User-Agent"] = c.http.Header().Get("User-Agent")
+
+	events := make(chan *sse.Event)
+	err := client.SubscribeChanRaw(events)
+	if err != nil {
+		return err
+	}
+	defer client.Unsubscribe(events)
+
+	for msg := range events {
+		if len(msg.Data) == 0 {
+			continue
+		}
+
+		if print {
+			err = c.printEntry(msg.Data, opts)
+		} else {
+			err = c.dumpEntry(msg.Data)
+		}
+
+		if err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+func (c *Client) dumpEntry(rawEntry []byte) error {
+	var entry Entry
+	err := json.Unmarshal(rawEntry, &entry)
+	if err != nil {
+		return err
+	}
+	fmt.Fprint(c.out, entry.Fields.Message)
```

Comment:
For the record I'm not a big fan of this approach, if someone uses the client with `os.Stdout` for example, they would get all the entries next to each other without new lines separator. There should be a well-defined API for following logs, eg. using a channel instead of an io.Writer.